front/src/store/index.js

```diff
@@ -50,11 +60,42 @@ export default new Vuex.Store({
     },
     logout ({ commit }) {
       commit('CLEAR_USER_DATA');
+    },
+    recoveryPasswordSendEmail ({ commit }, payload) {
+      return new Promise((resolve, reject) => {
+        axios.post(`${serverPath}api/ForgotPassword`,
+          { Email: payload.email }).then(resp => {
+          if (resp.status !== 200) {
+            reject(resp.data);
+          } else {
+            commit('SET_EMAIL_DATA', payload.email);
+            resolve();
+          }
+        });
+      });
+    },
+    recoveryPasswordSendPassword ({ commit, state }, passwordandtoken) {
+      return new Promise((resolve, reject) => {
+        axios.post(`${serverPath}api/ForgotPassword/ResetPassword`,
+          { Email: state.email, Password: passwordandtoken.password }, {
+            headers: { Authorization: 'Bearer ' + passwordandtoken.token }
```

Comment:
do we really need this header in "recovery" operation? seems it breaks security rules

Reply:
I fully agree that this violates security, it really needs to be redone at the root (it would be good to store the token in the database, but it is impossible to track that this token does not sit in the database for all eternity)

if necessary, I will redo it